client/src/hooks/Author/useSearchedAuthors.js

```diff
@@ -0,0 +1,16 @@
+import { ref, computed } from "vue";
+
+export default function useSearchedAuthors(authors) {
+  const searchQuery = ref("");
+  const searchedAuthors = computed(() => {
+    return authors.value.filter(
+      (item) =>
+        item.first_name
+          ?.toLowerCase()
```

Comment:
Looks like `?.` typescript syntax in js. Would be cool to change js to ts and add types to arguments.